edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -61,10 +52,10 @@ lazy_static! {
 
 #[derive(Clone)]
 pub struct DockerModuleRuntime {
-    client: DockerClient<UrlConnector>,
+    client: DockerClient,
     system_resources: Arc<Mutex<System>>,
     notary_registries: BTreeMap<String, PathBuf>,
-    notary_lock: tokio::sync::lock::Lock<BTreeMap<String, String>>,
+    notary_lock: Arc<tokio::sync::Mutex<BTreeMap<String, String>>>,
```

Comment:
I've been using `futures_util::lock::Mutex`. Though TBH not sure it matters.

Reply:
hmm, I don't have futures_util as a dep, so I'll just leave it as tokio rather than pull in a new dep